money.go

```diff
@@ -13,26 +25,72 @@ type Money struct {
 	Label          string  `json:"label" bson:"label"`
 	Dollars        float64 `json:"dollars" bson:"dollars"`
 
-	money *gomoney.Money
+	roundingMode string
+	money        *gomoney.Money
 }
 
-func New(cents int64, isoCode string) *Money {
+func New(cents int64, isoCode string, roundingMode string) *Money {
 	nm := gomoney.New(cents, isoCode)
 	return &Money{
 		money:          nm,
 		Cents:          cents,
 		Dollars:        nm.AsMajorUnits(),
 		CurrencyIso:    isoCode,
 		CurrencySymbol: nm.Currency().Grapheme,
+		roundingMode:   roundingMode,
 	}
 }
 
+// Setting the roundingMode of the money object
+func (m *Money) SetRoundingMode(mode string) {
+	m.roundingMode = mode
+}
+
+// Getting the roundingMode of the money object
+func (m *Money) GetRoundingMode() string {
+	return m.roundingMode
+}
+
 func (m *Money) initMoney() {
 	if m.money == nil {
 		m.money = gomoney.New(m.Cents, m.CurrencyIso)
 	}
 }
 
+// Getting the roound mode among all Money. If om is not exist, will return the first non-nil round mode. Otherwise will return ""
+func (m *Money) getRoundingModeAmongMoneys(ma []*Money) string {
```

Comment:
maybe rename to `alignRoundingMode` and re-assign `m.roundingMode` within this function